processing file asmFiles/test.loadstore.asm (1 of 1)
diff: memcpu.hex: No such file or directory
processing file asmFiles/count_days.asm (1 of 23)
diff: memcpu.hex: No such file or directory
processing file asmFiles/fib.asm (2 of 23)
diff: memcpu.hex: No such file or directory
processing file asmFiles/mergesort.asm (3 of 23)
--- memsim.hex	2023-02-18 20:47:56.422979854 -0500
+++ memcpu.hex	2023-02-18 20:47:56.298975213 -0500
@@ -81,133 +81,69 @@
 :040050000800004A5A
 :0400510003E00008C0
 :0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
-:0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
-:043FFE0000000500BA
+:0400C1000000005AE1
+:0400C20000000051E9
+:0400C3000000003306
+:0400C400000000191F
+:0400C50000000050E7
+:0400C600000000290D
+:0400C700000000161F
+:0400C800000000151F
+:0400C9000000000C27
+:0400CA000000003EF4
+:0400CB000000004BE6
+:0400CC0000000047E9
+:0400CD0000000053DC
+:0400CE0000000051DD
+:0400CF000000004DE0
+:0400D0000000001616
+:0400D1000000000B20
+:0400D2000000001D0D
+:0400D3000000000722
+:0400D4000000002107
+:0400D50000000063C4
+:0400D6000000001B0B
+:0400D70000000064C1
+:0400D80000000042E2
+:0400D9000000003DE6
+:0400DA000000002002
+:0400DB000000000120
+:0400DC0000000036EA
+:0400DD00000000041B
+:0400DE000000003DE1
+:0400DF0000000038E5
+:0400E0000000000319
+:0400E10000000030EB
+:0400E2000000000812
+:0400E30000000042D7
+:0400E40000000064B4
+:0400E5000000000F08
+:0400E6000000005CBA
+:0400E70000000041D4
+:0400E80000000020F4
+:0400E900000000090A
+:0400EA000000002FE3
+:0400EB0000000059B8
+:0400EC0000000011FF
+:0400ED000000000708
+:0400EE0000000023EB
+:0400EF0000000044C9
+:0400F00000000020EC
+:0400F1000000000A01
+:0400F2000000000703
+:0400F30000000017F2
+:0400F4000000005CAC
+:0400F5000000005BAC
+:0400F60000000028DE
+:0400F7000000001AEB
+:0400F80000000008FC
+:0400F90000000024DF
+:0400FA0000000026DC
+:0400FB0000000008F9
+:0400FC0000000026DA
+:0400FD0000000010EF
+:0400FE0000000032CC
+:0400FF0000000007F6
+:0401000000000043B8
+:043FFF0000000500B9
 :00000001FF
processing file asmFiles/mult.asm (4 of 23)
--- memsim.hex	2023-02-18 20:47:59.472093949 -0500
+++ memcpu.hex	2023-02-18 20:47:56.298975213 -0500
@@ -1,23 +1,149 @@
-:04000000341DFFFCB0
-:0400010034080003BC
-:0400020027BDFFFC1B
-:04000300AFA80000A2
-:0400040034090003B8
-:0400050027BDFFFC18
-:04000600AFA900009E
-:0400070008000008E5
-:0400080034020000BE
-:040009008FA30000C1
-:04000A0027BD00040A
-:04000B008FA40000BE
-:04000C0027BD000408
-:04000D00106000037C
-:04000E002063FFFF6D
-:04000F00008210203B
-:040010000800000DD7
-:0400110027BDFFFC0C
-:04001200AFA2000099
-:04001300FFFFFFFFED
-:043FFD0000000003BD
-:043FFE0000000009B6
+:04000000341EFFFCAF
+:04000100341DFFFCAF
+:0400020034040304BB
+:040003008C1003005A
+:0400040034090001BA
+:040005000130280698
+:040006000004882545
+:04000700000590253B
+:040008000C00001DCB
+:0400090034090001B5
+:04000A00013040067B
+:04000B00020828239C
+:04000C0034090002B1
+:04000D000128400482
+:04000E0034040304AF
+:04000F000088202124
+:04001000000498252B
+:040011000005A02521
+:040012000C00001DC1
+:040013000011202593
+:040014000012282589
+:04001500001330257F
+:040016000014382575
+:0400170034080500A4
+:0400180027BDFFFC05
+:04001900AFA800008C
+:04001A000C00002FA7
+:04001B0027BD0004F9
+:04001C00FFFFFFFFE4
+:04001D00340800049F
+:04001E00340A00029E
+:04001F00014548044B
+:040020000109082B9F
+:040021001020000C9F
+:040022000088C82169
+:040023008F38000012
+:04002400132400069B
+:040025008F2FFFFC1E
+:04002600030F082A92
+:0400270010200003A2
+:04002800AF2F0000F6
+:040029002739FFFC78
+:04002A0008000024A6
+:04002B00AF380000EA
+:04002C00250800049F
+:04002D0008000020A7
+:04002E0003E00008E3
+:04002F008FA8000096
+:0400300014A0000216
+:0400310014E00001D6
+:040032000800005171
+:0400330010E0000FCA
+:0400340010A0001503
+:040035008C890000B2
+:040036008CCA000070
+:04003700012A082A68
+:04003800102000058F
+:04003900AD0900000D
+:04003A002508000491
+:04003B002484000415
+:04003C0024A5FFFFF9
+:04003D000800003087
+:04003E00AD0A000007
+:04003F00250800048C
+:0400400024C60004CE
+:0400410024E7FFFFB2
+:040042000800003082
+:040043008C890000A4
+:04004400AD09000002
+:040045002508000486
+:0400460024A5FFFFEF
+:040047002484000409
+:0400480010A00008FC
+:040049000800004368
+:04004A008CCA00005C
+:04004B00AD0A0000FA
+:04004C00250800047F
+:04004D0024E7FFFFA6
+:04004E0024C60004C0
+:04004F0010E00001BC
+:040050000800004A5A
+:0400510003E00008C0
+:0400C00000000040FC
+:0400C1000000005AE1
+:0400C20000000051E9
+:0400C3000000003306
+:0400C400000000191F
+:0400C50000000050E7
+:0400C600000000290D
+:0400C700000000161F
+:0400C800000000151F
+:0400C9000000000C27
+:0400CA000000003EF4
+:0400CB000000004BE6
+:0400CC0000000047E9
+:0400CD0000000053DC
+:0400CE0000000051DD
+:0400CF000000004DE0
+:0400D0000000001616
+:0400D1000000000B20
+:0400D2000000001D0D
+:0400D3000000000722
+:0400D4000000002107
+:0400D50000000063C4
+:0400D6000000001B0B
+:0400D70000000064C1
+:0400D80000000042E2
+:0400D9000000003DE6
+:0400DA000000002002
+:0400DB000000000120
+:0400DC0000000036EA
+:0400DD00000000041B
+:0400DE000000003DE1
+:0400DF0000000038E5
+:0400E0000000000319
+:0400E10000000030EB
+:0400E2000000000812
+:0400E30000000042D7
+:0400E40000000064B4
+:0400E5000000000F08
+:0400E6000000005CBA
+:0400E70000000041D4
+:0400E80000000020F4
+:0400E900000000090A
+:0400EA000000002FE3
+:0400EB0000000059B8
+:0400EC0000000011FF
+:0400ED000000000708
+:0400EE0000000023EB
+:0400EF0000000044C9
+:0400F00000000020EC
+:0400F1000000000A01
+:0400F2000000000703
+:0400F30000000017F2
+:0400F4000000005CAC
+:0400F5000000005BAC
+:0400F60000000028DE
+:0400F7000000001AEB
+:0400F80000000008FC
+:0400F90000000024DF
+:0400FA0000000026DC
+:0400FB0000000008F9
+:0400FC0000000026DA
+:0400FD0000000010EF
+:0400FE0000000032CC
+:0400FF0000000007F6
+:0401000000000043B8
+:043FFF0000000500B9
 :00000001FF
processing file asmFiles/mult_procedure.asm (5 of 23)
--- memsim.hex	2023-02-18 20:48:02.143193900 -0500
+++ memcpu.hex	2023-02-18 20:47:56.298975213 -0500
@@ -1,32 +1,149 @@
-:04000000341DFFFCB0
-:0400010034080002BD
-:0400020027BDFFFC1B
-:04000300AFA80000A2
-:0400040034080003B9
-:0400050027BDFFFC18
-:04000600AFA800009F
-:0400070034080004B5
-:0400080027BDFFFC15
-:04000900AFA800009C
-:04000A003409FFFCBA
-:04000B002128FFFCAD
-:04000C00111D0002C0
-:04000D000C000010D3
-:04000E000800000ADC
-:04000F00FFFFFFFFF1
-:0400100034020000B6
-:040011008FA30000B9
-:0400120027BD000402
-:040013008FA40000B6
-:0400140027BD000400
-:040015001060000374
-:040016002063FFFF65
-:040017000082102033
-:0400180008000015C7
-:0400190027BDFFFC04
-:04001A00AFA2000091
-:04001B0003E00008F6
-:043FFC0000000004BD
-:043FFD000000000CB4
-:043FFE0000000018A7
+:04000000341EFFFCAF
+:04000100341DFFFCAF
+:0400020034040304BB
+:040003008C1003005A
+:0400040034090001BA
+:040005000130280698
+:040006000004882545
+:04000700000590253B
+:040008000C00001DCB
+:0400090034090001B5
+:04000A00013040067B
+:04000B00020828239C
+:04000C0034090002B1
+:04000D000128400482
+:04000E0034040304AF
+:04000F000088202124
+:04001000000498252B
+:040011000005A02521
+:040012000C00001DC1
+:040013000011202593
+:040014000012282589
+:04001500001330257F
+:040016000014382575
+:0400170034080500A4
+:0400180027BDFFFC05
+:04001900AFA800008C
+:04001A000C00002FA7
+:04001B0027BD0004F9
+:04001C00FFFFFFFFE4
+:04001D00340800049F
+:04001E00340A00029E
+:04001F00014548044B
+:040020000109082B9F
+:040021001020000C9F
+:040022000088C82169
+:040023008F38000012
+:04002400132400069B
+:040025008F2FFFFC1E
+:04002600030F082A92
+:0400270010200003A2
+:04002800AF2F0000F6
+:040029002739FFFC78
+:04002A0008000024A6
+:04002B00AF380000EA
+:04002C00250800049F
+:04002D0008000020A7
+:04002E0003E00008E3
+:04002F008FA8000096
+:0400300014A0000216
+:0400310014E00001D6
+:040032000800005171
+:0400330010E0000FCA
+:0400340010A0001503
+:040035008C890000B2
+:040036008CCA000070
+:04003700012A082A68
+:04003800102000058F
+:04003900AD0900000D
+:04003A002508000491
+:04003B002484000415
+:04003C0024A5FFFFF9
+:04003D000800003087
+:04003E00AD0A000007
+:04003F00250800048C
+:0400400024C60004CE
+:0400410024E7FFFFB2
+:040042000800003082
+:040043008C890000A4
+:04004400AD09000002
+:040045002508000486
+:0400460024A5FFFFEF
+:040047002484000409
+:0400480010A00008FC
+:040049000800004368
+:04004A008CCA00005C
+:04004B00AD0A0000FA
+:04004C00250800047F
+:04004D0024E7FFFFA6
+:04004E0024C60004C0
+:04004F0010E00001BC
+:040050000800004A5A
+:0400510003E00008C0
+:0400C00000000040FC
+:0400C1000000005AE1
+:0400C20000000051E9
+:0400C3000000003306
+:0400C400000000191F
+:0400C50000000050E7
+:0400C600000000290D
+:0400C700000000161F
+:0400C800000000151F
+:0400C9000000000C27
+:0400CA000000003EF4
+:0400CB000000004BE6
+:0400CC0000000047E9
+:0400CD0000000053DC
+:0400CE0000000051DD
+:0400CF000000004DE0
+:0400D0000000001616
+:0400D1000000000B20
+:0400D2000000001D0D
+:0400D3000000000722
+:0400D4000000002107
+:0400D50000000063C4
+:0400D6000000001B0B
+:0400D70000000064C1
+:0400D80000000042E2
+:0400D9000000003DE6
+:0400DA000000002002
+:0400DB000000000120
+:0400DC0000000036EA
+:0400DD00000000041B
+:0400DE000000003DE1
+:0400DF0000000038E5
+:0400E0000000000319
+:0400E10000000030EB
+:0400E2000000000812
+:0400E30000000042D7
+:0400E40000000064B4
+:0400E5000000000F08
+:0400E6000000005CBA
+:0400E70000000041D4
+:0400E80000000020F4
+:0400E900000000090A
+:0400EA000000002FE3
+:0400EB0000000059B8
+:0400EC0000000011FF
+:0400ED000000000708
+:0400EE0000000023EB
+:0400EF0000000044C9
+:0400F00000000020EC
+:0400F1000000000A01
+:0400F2000000000703
+:0400F30000000017F2
+:0400F4000000005CAC
+:0400F5000000005BAC
+:0400F60000000028DE
+:0400F7000000001AEB
+:0400F80000000008FC
+:0400F90000000024DF
+:0400FA0000000026DC
+:0400FB0000000008F9
+:0400FC0000000026DA
+:0400FD0000000010EF
+:0400FE0000000032CC
+:0400FF0000000007F6
+:0401000000000043B8
+:043FFF0000000500B9
 :00000001FF
processing file asmFiles/search.asm (6 of 23)
--- memsim.hex	2023-02-18 20:48:05.425316715 -0500
+++ memcpu.hex	2023-02-18 20:48:05.333313272 -0500
@@ -14,7 +14,7 @@
 :04000D001000FFF9E7
 :04000E00AFA500009A
 :04000F00FFFFFFFFF1
-:0400200000000124B7
+:040020000000008C50
 :0400210000005C6F10
 :040022000000006476
 :040023000000087D54
processing file asmFiles/test.allcommands.asm (7 of 23)
--- memsim.hex	2023-02-18 20:48:09.142455808 -0500
+++ memcpu.hex	2023-02-18 20:48:05.333313272 -0500
@@ -1,49 +1,120 @@
-:040000003403002E97
-:04000100340400576C
-:04000200340100F0D5
-:040003000064102065
-:04000400AC2200002A
-:040005000064102162
-:04000600AC22000424
-:04000700006410245D
-:04000800AC2200081E
-:040009000064102758
-:04000A00AC22001014
-:04000B000064102558
-:04000C00AC2200140E
-:04000D000064102A51
-:04000E00AC22001808
-:04000F000064102B4E
-:04001000AC22001C02
-:040011000064100473
-:04001200AC220020FC
-:04001300006410066F
-:04001400AC220024F6
-:040015000064102251
-:04001600AC220028F0
-:04001700006410234E
-:04001800AC22002CEA
-:040019000064102649
-:04001A00AC220030E4
-:04001B0020620064FB
-:04001C00AC220034DE
-:04001D0024620064F5
-:04001E00AC220038D8
-:04001F0030620064E7
-:04002000AC22003CD2
-:04002100FFFFFFFFDF
-:04003C00000000853B
-:04003D00000000853A
-:04003E0000000006B8
-:04004000FFFFFF803F
-:040041000000007F3C
-:0400420000000001B9
-:0400430000000001B8
-:040044000015C000E3
-:04004600FFFFFFD7E2
-:04004700FFFFFFD7E1
-:04004800000000793B
-:040049000000009221
-:04004A000000009220
-:04004B00000000248D
+:04000000341D00802B
+:0400010034010001C5
+:0400020034020004C0
+:04000300AFA00000AA
+:040004008FA30004C2
+:040005008FA40008BC
+:0400060027A5000C1E
+:040007008CA60000C3
+:0400080000C33823D6
+:0400090010E00004FF
+:04000A0000A2282107
+:04000B00008120232D
+:04000C00108000025E
+:04000D001000FFF9E7
+:04000E00AFA500009A
+:04000F00FFFFFFFFF1
+:040020000000008C50
+:0400210000005C6F10
+:040022000000006476
+:040023000000087D54
+:0400240000005FCBAE
+:040025000000A41A19
+:04002600000041098C
+:04002700000045226E
+:040028000000700F55
+:040029000000766DF0
+:04002A0000006F6003
+:04002B0000008A5EE9
+:04002C0000009580BB
+:04002D00000070A3BC
+:04002E000000AEA977
+:04002F000000711A42
+:0400300000006F81DC
+:0400310000008F9AA2
+:040032000000258421
+:040033000000A5998B
+:040034000000401573
+:040035000000CE8178
+:040036000000F55B76
+:040037000000399EEE
+:040038000000A23FE3
+:040039000000358806
+:04003A00000033ACE3
+:04003B000000BCE71E
+:04003C0000002A6B2B
+:04003D0000009FA17F
+:04003E000000C94BAA
+:04003F000000C65B9C
+:040040000000006854
+:040041000000F4992E
+:0400420000005F71EA
+:040043000000D06F7A
+:04004400000014DFC5
+:040045000000116541
+:040046000000F88D31
+:0400470000004BA4C6
+:0400480000002E7412
+:0400490000005C6FE8
+:04004A000000D11EC3
+:04004B0000009222FD
+:04004C000000ACDB29
+:04004D000000103867
+:04004E000000AB17EC
+:04004F000000F7CEE8
+:0400500000008A9E84
+:0400510000009AA36E
+:040052000000B49561
+:0400530000008A5EC1
+:040054000000D85977
+:0400550000000BACF0
+:040056000000D0DBFB
+:04005700000035521E
+:040058000000A6B04E
+:040059000000727FB2
+:04005A00000028E496
+:04005B000000E5CFED
+:04005C000000163C4E
+:04005D00000034115A
+:04005E0000008F0708
+:04005F000000FAB7EC
+:0400600000000F3459
+:040061000000DABF02
+:0400620000006F6FBC
+:040063000000C5983C
+:040064000000F4960E
+:0400650000009A9A63
+:040066000000BD6A6F
+:04006700000021363E
+:040068000000810A09
+:040069000000CA5574
+:04006A0000008BCE39
+:04006B0000002AC4A3
+:04006C000000DDCEE5
+:04006D000000DD06AC
+:04006E000000C4FCCE
+:04006F000000FB2F63
+:040070000000EE5F3F
+:040071000000FD305E
+:040072000000C54085
+:040073000000D5F1C3
+:040074000000BDAD1E
+:04007500000045C37F
+:040076000000708A8C
+:040077000000A35989
+:040078000000F40D83
+:040079000000BA06C3
+:04007A000000BACEFA
+:04007B000000B44786
+:04007C0000003F48F9
+:04007D000000899E58
+:04007E00000080847A
+:04007F000000BDB907
+:040080000000A05A82
+:040081000000E22574
+:040082000000FB0C73
+:040083000000B2B215
+:040084000000A4DBF9
+:0400850000008BF9F3
+:04008600000012F76D
 :00000001FF
processing file asmFiles/test.beq_nt.asm (8 of 23)
processing file asmFiles/test.beq_t.asm (9 of 23)
processing file asmFiles/test.bne_nt.asm (10 of 23)
processing file asmFiles/test.bne_t.asm (11 of 23)
processing file asmFiles/test.branchjump1.asm (12 of 23)
processing file asmFiles/test.data_hazard.asm (13 of 23)
--- memsim.hex	2023-02-18 20:48:33.078351473 -0500
+++ memcpu.hex	2023-02-18 20:48:32.970347431 -0500
@@ -12,7 +12,7 @@
 :04001B00004320205E
 :04001F00AC0400F835
 :04002000FFFFFFFFE0
-:04003C0000000006BA
-:04003D0000000006B9
-:04003E0000000010AE
+:04003C0000000004BC
+:04003D0000000004BB
+:04003E000000000AB4
 :00000001FF
processing file asmFiles/test.halt.asm (14 of 23)
--- memsim.hex	2023-02-18 20:48:35.782452658 -0500
+++ memcpu.hex	2023-02-18 20:48:35.631447008 -0500
@@ -4,5 +4,4 @@
 :04000300FFFFFFFFFD
 :0400040034080BAD04
 :04000500AC0800F053
-:04003C000000ABCD48
 :00000001FF
processing file asmFiles/test.jal.asm (15 of 23)
processing file asmFiles/test.j.asm (16 of 23)
processing file asmFiles/test.jr.asm (17 of 23)
--- memsim.hex	2023-02-18 20:48:48.189916937 -0500
+++ memcpu.hex	2023-02-18 20:48:48.106913831 -0500
@@ -7,5 +7,5 @@
 :040006003408ABCD42
 :04000700AC0800F051
 :04000800FFFFFFFFF8
-:04003C000000ABCD48
+:04003C0000000018A8
 :00000001FF
processing file asmFiles/test.jump.asm (18 of 23)
--- memsim.hex	2023-02-18 20:48:51.484040201 -0500
+++ memcpu.hex	2023-02-18 20:48:51.398036983 -0500
@@ -7,5 +7,4 @@
 :040006003402F0F0E0
 :04000700AC22000027
 :04000800FFFFFFFFF8
-:04003C000000F0F0E0
 :00000001FF
processing file asmFiles/test.ldst1.asm (19 of 23)
--- memsim.hex	2023-02-18 20:48:54.513153548 -0500
+++ memcpu.hex	2023-02-18 20:48:51.398036983 -0500
@@ -1,21 +1,10 @@
-:04000000340100F0D7
-:0400010034020100C4
-:0400020034030200C1
-:0400030034040300BE
-:0400040034050400BB
-:040005008C26000045
-:040006008C2700043F
-:040007008C28000839
-:0400080034040500B7
-:0400090034050600B4
-:04000A00AC46000000
-:04000B00AC470004FA
-:04000C00AC480008F4
-:04000D00FFFFFFFFF3
-:04003C000000733716
-:04003D000000270197
-:04003E000000133774
-:040040000000733712
-:040041000000270193
-:040042000000133770
+:04000000341DFFFCB0
+:04000100340100F0D6
+:0400020008000006EC
+:0400030034020BAD0B
+:04000400AC2200002A
+:04000500FFFFFFFFFB
+:040006003402F0F0E0
+:04000700AC22000027
+:04000800FFFFFFFFF8
 :00000001FF
processing file asmFiles/test.ldst2.asm (20 of 23)
--- memsim.hex	2023-02-18 20:48:57.978283211 -0500
+++ memcpu.hex	2023-02-18 20:48:51.398036983 -0500
@@ -1,20 +1,10 @@
-:040000003C0AFEEDCB
-:0400010034010F00B7
-:0400020034020800BC
-:04000300354ABEEFCD
-:040004008C23000049
-:040005008C24000443
-:040006008C2500083D
-:04000700AC43000006
-:04000800AC44000400
-:04000900AC450008FA
-:04000A00AC4A000CF0
-:04000B00FFFFFFFFF5
-:040200000000733750
-:0402010000002701D1
-:0402020000001337AE
-:04020300FEEDBEEF5F
-:0403C000000073378F
-:0403C1000000270110
-:0403C20000001337ED
+:04000000341DFFFCB0
+:04000100340100F0D6
+:0400020008000006EC
+:0400030034020BAD0B
+:04000400AC2200002A
+:04000500FFFFFFFFFB
+:040006003402F0F0E0
+:04000700AC22000027
+:04000800FFFFFFFFF8
 :00000001FF
processing file asmFiles/test.load_hazard.asm (21 of 23)
--- memsim.hex	2023-02-18 20:49:00.903392666 -0500
+++ memcpu.hex	2023-02-18 20:49:00.740386567 -0500
@@ -9,5 +9,4 @@
 :04000800000838258F
 :04000900AC0700F050
 :04000A00FFFFFFFFF6
-:04003C000000ABCD48
 :00000001FF
processing file asmFiles/test.loadstore.asm (22 of 23)
--- memsim.hex	2023-02-18 20:49:04.106512525 -0500
+++ memcpu.hex	2023-02-18 20:49:00.740386567 -0500
@@ -1,20 +1,12 @@
-:04000000340100F0D7
-:040001003402008045
-:040002003C07DEAD2C
-:0400030034E7BEEF31
-:040004008C23000049
-:040005008C24000443
-:040006008C2500083D
-:04000700AC43000006
-:04000800AC44000400
-:04000900AC450008FA
-:04000A00AC47000CF3
-:04000B00FFFFFFFFF5
-:040020000000733732
-:0400210000002701B3
-:040022000000133790
-:04002300DEADBEEFA1
-:04003C000000733716
-:04003D000000270197
-:04003E000000133774
+:04000000341DFFFCB0
+:040001003408ABCD47
+:04000200AC0800F056
+:0400030034080BAD05
+:040004008C0800F074
+:040005000008382592
+:040006008C0800F072
+:040007000007302599
+:04000800000838258F
+:04000900AC0700F050
+:04000A00FFFFFFFFF6
 :00000001FF
processing file asmFiles/test.rtype.asm (23 of 23)
--- memsim.hex	2023-02-18 20:49:07.843652366 -0500
+++ memcpu.hex	2023-02-18 20:49:00.740386567 -0500
@@ -1,41 +1,12 @@
-:040000003401D2698C
-:04000100340237F19D
-:040002003415008031
-:04000300341600F0BF
-:040004000022182599
-:040005000022202491
-:040006003025000F92
-:040007000022302182
-:0400080024678740A2
-:04000900008240230E
-:04000A0000A24826E2
-:04000B00382AF33F5D
-:04000C00340E0004AA
-:04000D0001C15804D1
-:04000E00340E0005A7
-:04000F0001C16006C5
-:04001000002268273B
-:04001100AECD000070
-:04001200AEA3000099
-:04001300AEA4000493
-:04001400AEA500088D
-:04001500AEA6000C87
-:04001600AEA7001081
-:04001700AEA800147B
-:04001800AEA9001875
-:04001900AEAA001C6F
-:04001A00AEAB002069
-:04001B00AEAC002463
-:04001C00FFFFFFFFE4
-:040020000000F7F9EC
-:040021000000126168
-:0400220000000009D1
-:0400230000010A5A74
-:0400240000007F3920
-:04002500FFFFDA708F
-:04002600000037F8A7
-:04002700000021565E
-:04002800000D269011
-:04002900000006933A
-:04003C00FFFF0806B4
+:04000000341DFFFCB0
+:040001003408ABCD47
+:04000200AC0800F056
+:0400030034080BAD05
+:040004008C0800F074
+:040005000008382592
+:040006008C0800F072
+:040007000007302599
+:04000800000838258F
+:04000900AC0700F050
+:04000A00FFFFFFFFF6
 :00000001FF
